README.md

```diff
@@ -1,6 +1,28 @@
 # Portfolio
-I am a part owner of a carpet cleaning company. I wanted to be able to analyze our biometrics to get a feel for how 
-my technicians were doing with additional services. As well as which cities or zipcodes seemed to make more money.
-This Portfolio comtains tables I have created as well as stored procedures created with SQL in order to Extract,
-Transform and Load the data I entered into a spreadsheet. This can be used for most any carpet cleaning company
-as well as altered for use in other services industries.
+<img src="https://user-images.githubusercontent.com/108885804/210106590-9f1c0477-dc49-4700-a4b3-0ef3154a7e87.jpg" height="200px"/>
+
+My name is Rhet Pulley. I am a part owner of a carpet cleaning company. I wanted to be able to analyze our biometrics to get a feel for how 
```

Comment:
Forget about carpet cleaning, move straight into your credentials, that you're a data analyst, SQL developer, report writer with knowledge of ETL processes etc.